### DISCUSSION TITLE
Twitter login works, however where's oauth_token and oauth_token_secret

### DISCUSSION THREAD

**Discussion by spidgorny** [nextauthjs/next-auth#1723] [Help] [answered]
My Twitter login is working fine.
My next step is to use https://www.npmjs.com/package/twitter-v2 to query user data with ```access_token_key``` and 
```access_token_secret```. These are not available after login. I only have ```accessToken```.

```
account: {
    provider: 'twitter',
    type: 'oauth',
    id: 'xxx',
    accessToken: 'xxx',
    refreshToken: 'xxx',
    results: [Object: null prototype] {
      user_id: 'xxx',
      screen_name: 'xxx'
    }
  }
```

Is there a way to query Twitter API on behalf of the user who has just logged-in?

**Comment by balazsorban44** [comment, marked as answer]
#1322 should fix this soon!

**Reply by balazsorban44** [reply]
Like now... It is available now, under `oauth_token` and `oauth_token_secret`! 🎉

**Reply by spidgorny** [reply]
Wow, you are fast 😱

**Reply by balazsorban44** [reply]
Please let me know if it works correctly!
